test/parser/parser_integration_test.exs

```diff
@@ -0,0 +1,80 @@
+defmodule Membrane.H264.FFmpeg.Parser.IntegrationTest do
+  use ExUnit.Case
+  import Membrane.ParentSpec
+  import Membrane.Testing.Assertions
+
+  alias Membrane.Testing
+
+  @fixtures_dir "./test/fixtures/"
+  @stream_with_params File.read!("test/fixtures/input-10-720p.h264")
```

Comment:
I think we should be feeding the stream without the parameters in order to actually test anything